File: Data_Structure/tests/TestLinkedList.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestLinkedList {
    @Test
    public void testAppend() {
        LinkedList ll = new LinkedList();
        ll.append(1);
        ll.append(2);
        assertEquals(1, ll.head.value);
        assertEquals(2, ll.head.next.value);
    }
}

File: Data_Structure/tests/test_linkedlist.py
import unittest
from Python.LinkedList import LinkedList

class TestLinkedList(unittest.TestCase):
    def test_append(self):
        ll = LinkedList()
        ll.append(1)
        ll.append(2)
        self.assertEqual(ll.head.value, 1)
        self.assertEqual(ll.head.next.value, 2)

if __name__ == "__main__":
    unittest.main()
